doc/en/monkeypatch.rst

```diff
@@ -110,7 +286,7 @@ both paths can be safely tested without impacting the running environment:
 
 .. code-block:: python
 
-    # contents of our test file e.g. test_code.py
+    # Contents of our test file e.g. test_code.py.
```

Comment:
Also, if I'm being pedantic, we usually don't use `.` at the end of single line comments

Reply:
I changed this throughout the file with regard to the single line comments.